### DISCUSSION TITLE
How to access Lambda context from Rust handler?

### DISCUSSION THREAD

**Discussion by Samrose-Ahmed** [smithy-lang/smithy-rs#2628] [Q&A]
I may be missing this in the docs but how does one access the lambda context from a handler?

**Comment by ysaito1001** [comment]
Hi Samrose,

Thank you for posting a question. ~~Could you describe more about your use case and a handler?~~ Could you repost the question in the [aws-lambda-rust-runtime](https://github.com/awslabs/aws-lambda-rust-runtime) who can better assist you?
